### PR TITLE
PIE-564: updated text on dough staking campaign page

### PR DESCRIPTION
**Pull request by TotoFullStackDev**
##### Checklist

- [ ] Linter status: 100% pass
- [ ] Changes don't break existing behavior
- [ ] Test coverage hasn't decreased

### CODE REVIEW

**Review by AdrianAddesso — CHANGES_REQUESTED** [review]
There's a typo: Distsributed

Please do change that whole sentece to 'Distributed in the first month!'

File: src/pages/DoughStakingCampaign.svelte [hunk 1]
```diff
@@ -150,11 +150,10 @@
     <img class="h-150px inline mb-4" src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/apple/285/money-mouth-face_1f911.png" alt="hourglass" />
     <div class="font-hero text-center mt-10">$ 1,350,000</div>
     <div class="font-thin text-l text-center mt-20px">
-      You are reading it right.
+      Distsributed during the first month distribution!
```
Comment (Alexintosh) [inline src/pages/DoughStakingCampaign.svelte]: typo

**Review by Alexintosh — CHANGES_REQUESTED** [review]

**Review by AdrianAddesso — CHANGES_REQUESTED** [review]
Still pending to update to:
'Distributed during the first month distribution!'
to
'Distributed in the first month!'

**Review by Alexintosh — APPROVED** [review]

**Review by Alexintosh — CHANGES_REQUESTED** [review]
I don't understand why we keep the 135000 text since it's over. More effort is required on this page to be updates.

**Review by AdrianAddesso — CHANGES_REQUESTED** [review]
Final copy update: see image as outcome example:
Title: Earn rewards in three easy steps
Body: Stake your DOUGH, vote on proposals and claim your monthly rewards. 250+ members joined in the first month! 
CTA: Stake DOUGH to start earning rewards


image

**Review by AdrianAddesso — APPROVED** [review]
Changes approved

**Review by dimarconicola — APPROVED** [review]
Looks good to me

**Review by dantop114 — APPROVED** [review]
lgtm

**Review by Alexintosh — APPROVED** [review]

**Pull request merged**